src/analysis/special_functions/logb.lean

```diff
@@ -0,0 +1,325 @@
+/-
+Copyright (c) 2022 Bolton Bailey. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bolton Bailey, Chris Hughes, Abhimanyu Pallavi Sudhir, Jean Lo, Calle Sönne
+-/
+import analysis.special_functions.log
+import analysis.special_functions.pow
+
+/-!
+# Real logarithm base `b`
+
+In this file we define `real.logb` to be the logarithm of a real number in a given base `b`. We
+define this as the division of the natural logarithms of the argument and the base, so that we have
+a globally defined function with `logb b 0 = 0`, `logb b (-x) = logb b x` `logb 0 x = 0` and
+`logb (-b) x = logb b x`.
+
+We prove some basic properties of this function and it's relation to `rpow`.
+
+## Tags
+
+logarithm, continuity
+-/
+
+open set filter function
+open_locale topological_space
+noncomputable theory
+
+namespace real
+
+variables {b x y : ℝ}
+
+/-- The real logarithm in a given base. As with the natural logarithm, we define `logb b x` to
+be `logb b |x|` for `x < 0`, and `0` for `x = 0`.-/
+@[pp_nodot] noncomputable def logb (b x : ℝ) : ℝ := log x / log b
+
+lemma log_div_log : log x / log b = logb b x := rfl
```

Comment:
I see that this is no longer a `simp` lemma. Then the API for `logb` can be as incomplete as you wish.

Reply:
I see. I wasn't understanding that the `simp` status was the critical factor in whether the API needed to be built out but in retrospect it makes sense.